From cedc20289917ff608bc895462d6813ea499e93eb Mon Sep 17 00:00:00 2001
From: Jeffy Chen <jeffy.chen@rock-chips.com>
Date: Fri, 19 Apr 2024 09:21:22 +0800
Subject: [PATCH 80/81] Revert "HACK: modesetting: Disable fallback probe when
 having other options"

This reverts commit 0563dc72745f6ccdc610e52e3bd497b20da8fcc6.

Specify primary GPU in xorg.conf instead:
Section "OutputClass"
    Identifier  "RockchipDRM"
    MatchDriver "rockchip"
    Option      "PrimaryGPU"     "yes"
EndSection

Signed-off-by: Jeffy Chen <jeffy.chen@rock-chips.com>
---
 hw/xfree86/drivers/modesetting/driver.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/hw/xfree86/drivers/modesetting/driver.c b/hw/xfree86/drivers/modesetting/driver.c
index 061647db4..904164c77 100644
--- a/hw/xfree86/drivers/modesetting/driver.c
+++ b/hw/xfree86/drivers/modesetting/driver.c
@@ -488,9 +488,6 @@ Probe(DriverPtr drv, int flags)
     const char *dev;
     ScrnInfoPtr scrn = NULL;
 
-    if (XSERVER_PLATFORM_BUS || XSERVER_LIBPCIACCESS)
-        return FALSE;
-
     /* For now, just bail out for PROBE_DETECT. */
     if (flags & PROBE_DETECT)
         return FALSE;
-- 
2.38.5

